packages/cli/src/commands/generate/page/page.js

```diff
@@ -100,47 +103,16 @@ export const routes = ({ name, path }) => {
   ]
 }
 
-export const command = 'page <name> [path]'
-export const description = 'Generate a page and route component'
-export const builder = (yargs) => {
-  yargs
-    .positional('name', {
-      description: 'Name of the page',
-      type: 'string',
-    })
-    .positional('path', {
-      description: 'URL path to the page, or just {param}. Defaults to name',
-      type: 'string',
-    })
-    .option('force', {
-      alias: 'f',
-      default: false,
-      description: 'Overwrite existing files',
-      type: 'boolean',
-    })
-    .option('tests', {
-      description: 'Generate test files',
-      type: 'boolean',
-      default: true,
-    })
-    .option('typescript', {
-      alias: ['ts'],
-      description: 'Generate in TypeScript',
-      type: 'boolean',
-      default: false,
-    })
-    .option('stories', {
-      description: 'Generate storybook files',
-      type: 'boolean',
-      default: true,
-    })
-    .epilogue(
-      `Also see the ${terminalLink(
-        'Redwood CLI Reference',
-        'https://redwoodjs.com/reference/command-line-interface#generate-page'
-      )}`
-    )
-}
+// @NOTE: Not exporting handler from function
+// As pages need a special handler
+export const {
+  command,
+  description,
+  builder,
+} = createYargsForComponentGeneration({
```

Comment:
@dac09 Will this still `Generate storybook files`?

Reply:
It does @dthyresson - it does yeah. This code does read a little convoluted, renamed a few variables